fs-sim/test/Test/System/FS/StateMachine.hs

```diff
@@ -147,7 +156,11 @@ data Cmd fp h =
   | Seek               h SeekMode Int64
   | Get                h Word64
   | GetAt              h Word64 AbsOffset
+  | GetBuf             h ByteCount
+  | GetBufAt           h ByteCount AbsOffset
```

Comment:
One thing I don't like from QSM is that you cannot tell which response is the one expected from each command. Perhaps some comments here might help?
```haskell
  | GetBuf   h ByteCount             -- -> ByteCount
  ...
  | PutBuf   h ByteString ByteCount  -- -> BCBS
```

Reply:
Added some comments